File: v2/android/src/main/java/io/swagger/client/model/Posts.java
package io.swagger.client.model;

import io.swagger.client.model.Post;
import java.util.*;

import com.wordnik.swagger.annotations.*;
import com.fasterxml.jackson.annotation.JsonProperty;


@ApiModel(description = "")
public class Posts  {
  
  private List<Post> posts = new ArrayList<Post>() ;

  
  /**
   **/
  @ApiModelProperty(value = "")
  @JsonProperty("posts")
  public List<Post> getPosts() {
    return posts;
  }
  public void setPosts(List<Post> posts) {
    this.posts = posts;
  }

  

  @Override
  public String toString()  {
    StringBuilder sb = new StringBuilder();
    sb.append("class Posts {\n");
    
    sb.append("  posts: ").append(posts).append("\n");
    sb.append("}\n");
    return sb.toString();
  }
}

File: v2/csharp/src/main/csharp/io/swagger/Model/Posts.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;

namespace io.swagger.Model {
  public class Posts {
    

    
    public List<Post> Posts { get; set; }

    

    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class Posts {\n");
      
      sb.Append("  Posts: ").Append(Posts).Append("\n");
      
      sb.Append("}\n");
      return sb.ToString();
    }
  }
  
  
}